HEN_HOUSE/doc/src/pirs898-egs++/egs_kerma.doxy

```diff
@@ -0,0 +1,562 @@
+    # simulations. See block 'scoring options'
+    # for more detail.
+    ##########################
+    :start geometry:
+        library = egs_genvelope
+        name = cavity_in_room_no_wall
+        base geometry = room
+        inscribed geometries = cavity
+    :stop geometry:
+    ##########################
+    # An air sphere in a room with concrete walls
+    # Wall contribution included during the
+    # simulations. See block 'scoring options'
+    # for more detail.
+    ##########################
+    :start geometry:
+        library = egs_genvelope
+        name = cavity_in_room_all
+        base geometry = room
+        inscribed geometries = cavity
+    :stop geometry:
+
+    simulation geometry = cavity_in_room_no_wall
+
+:stop geometry definition:
+
+:start source definition:
+########################################### define the source:
+   :start source:
+     library = egs_isotropic_source
+     name = the_source
+     :start shape:
+       type = point
+       position = 0, 0, -100
+     :stop shape:
+     :start spectrum:
+       type = monoenergetic
+       energy = .04
+     :stop spectrum:
+     charge = 0
+   :stop source:
+
+   simulation source = the_source
+
+:stop source definition:
+
+##################################### Run control
+:start run control:
+
+    ncase = 1000000
+    #calculation = combine
+
+:stop run control:
+
+##################################### Scoring options
+:start scoring options:
+
+    ###############################################
+    # Using same geometry under two diff. names for
+    # easier bookeeping.
+    ###############################################
+    :start calculation geometry:
+        geometry name = cavity_in_room_no_wall
+        cavity regions = 2
+        excluded regions = 0 # exclude particles passing through these regions
+        cavity mass = 0.630831804841 # 5 cm radius air sphere
+    :stop calculation geometry:
+    :start calculation geometry:
+        geometry name = cavity_in_room_all
+        cavity regions = 2
+        #excluded regions = 0 # exclude particles passing through these regions
+        cavity mass = 0.630831804841 # 5 cm radius air sphere
+    :stop calculation geometry:
+
+    # Ratio estimates wall contribution to air sphere
+    correlated geometries = cavity_in_room_all cavity_in_room_no_wall
+
+    ############################################
+    # Fluence scoring requested in this block
+    # Common to all calculation geometries
+    ############################################
+    :start fluence scoring:
+       minimum energy = 0.0
+       maximum energy = 0.040
+       number of bins = 400
+       scale          = linear
+    :stop fluence scoring:
+
+    ######################################
+    # E*muen file, it could also be E*mutr
+    #
+    # Replace $EGS_HOME with the actual path!
+    ######################################
+    emuen file = $EGS_HOME/egs_kerma/emuen_icru90_1.5MeV.data
```

Comment:
Will this work with a relative path so that they don't need to replace $EGS_HOME?